utils/cliutils/commandsflags.go

```diff
@@ -1049,6 +1060,18 @@ var flagsMap = map[string]cli.Flag{
 		Name:  audience,
 		Usage: "[Optional] A space-separate list of the other Artifactory instances or services that should accept this token identified by their Artifactory Service IDs, as obtained by the 'jfrog rt curl api/system/service_id' command.` `",
 	},
+	csv: cli.StringFlag{
+		Name:  csv,
+		Usage: "[Mandatory] Path to a csv file contains users details. The first row of the file should contain the user fields names e.g name,password,email` `",
+	},
+	usersGroups: cli.StringFlag{
+		Name:  usersGroups,
+		Usage: "[Optional] A list of comma-separated groups for the new users to be associated with.` `",
+	},
+	replaceExist: cli.BoolFlag{
+		Name:  replaceExist,
+		Usage: "[Optional] Set to true if you'd like the exist user or group to be replaced (overridded).` `",
```

Comment:
[Optional] Set to true if you'd like the exist user or group to be replaced (overridded).
-->
[Optional] Set to true if you'd like existing users or groups to be replaced.